core/modules/views/includes/view.inc

```diff
@@ -1224,6 +1224,7 @@ class view {
    */
   function render($display_id = NULL) {
     $this->execute($display_id);
+    $views_settings_config = config('views.settings');
```

Comment:
... ehm ... guess what:
```
$show_additional_queries = config_get('views.settings', 'show_additional_queries');
```
Only "show_additional_queries" is needed in this function.

Reply:
Absolutely right! Corrected.